.evergreen/config.yml

```diff
@@ -140,30 +177,20 @@ tasks:
     commands:
       - func: "validate executor"
   # One test-case per task.
-  - name: retryReads-resizeCluster
-    tags: ["all", "retryReads", "resizeCluster"]
-    commands:
-      - func: "run test"
-        vars:
-          TEST_NAME: retryReads-resizeCluster
-  - name: retryReads-toggleServerSideJS
-    tags: ["all", "retryReads", "toggleServerSideJS"]
-    commands:
-      - func: "run test"
-        vars:
-          TEST_NAME: retryReads-toggleServerSideJS
-  - name: retryWrites-resizeCluster
-    tags: ["all", "retryWrites", "resizeCluster"]
+  - name: retryReads-processRestart
+    cron: '@weekly'
+    tags: ["all"]
     commands:
       - func: "run test"
         vars:
-          TEST_NAME: retryWrites-resizeCluster
-  - name: retryWrites-toggleServerSideJS
-    tags: ["all", "retryWrites", "toggleServerSideJS"]
+          TEST_NAME: retryReads-processRestart
+  - name: retryReads-processRestart-sharded
+    cron: '@weekly'
+    tags: ["all"]
     commands:
       - func: "run test"
         vars:
-          TEST_NAME: retryWrites-toggleServerSideJS
```

Comment:
Is there a reason you aren't running the other tests? There seem to be around 10 test files but only 2 of them have been added to the configuration here.

Reply:
Changed the configuration to run all of the tests.